logicrunner/builtin/contract/deposit/deposit.go

```diff
@@ -29,7 +28,8 @@ import (
 type status string
 
 const (
-	confirms uint = 3
+	migrationConfirms       uint                = 3
```

Comment:
I think this not correct depositConfirm.  It is migration manager confirm. Correct short name migrationConfirms

Reply:
Dont use package name or semilar things in naming if it is not needed. If u left just **confirms** - it will be okay for readers. Also there is no other confirms, so dont add any prefix.

Reply:
isn't outdated